pkg/clients/helm/userinfo.go

```diff
@@ -0,0 +1,7 @@
+package helm
+
+// UserInfo keeps auth information to access a Helm Chart
+type UserInfo struct {
```

Comment:
nit: I would suggest naming this struct as `RepoCreds` or `PullCreds` with fields `Username` & `Password`

Reply:
sounds good!